examples/medplum-fsh-profiles/README.md

```diff
@@ -1,13 +1,22 @@
 # Medplum FSH Profiles
 
-This project contains FHIR Shorthand (FSH) definitions for custom FHIR profiles used with Medplum. It currently includes a basic Patient profile that requires a birth date.
+This project contains FHIR Shorthand (FSH) definitions for custom FHIR profiles used with Medplum. It includes patient and healthcare service profiles with custom vocabularies.
```

Comment:
```suggestion
This project contains FHIR Shorthand (FSH) definitions for custom FHIR profiles used with Medplum. It includes `Patient` and `HealthcareService` profiles with custom vocabularies.
```

Since profiles are tied to specific FHIR resources, we should follow the consistent style of: 
1. Backticks
2. Capitalization matching the FHIR spec